openstates/de/committees.py

```diff
@@ -22,29 +22,41 @@ def scrape(self, chamber, term):
 
         url = urls[chamber] % (session,)
```

Comment:
Joint committees in DE are really confusing. I have a call in to their leg to figure this out, and am going to put this on hold till they get back to me.